core/accounts/wallet.go

```diff
@@ -0,0 +1,24 @@
+package accounts
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+
+	"github.com/cometbft/cometbft/crypto/ed25519"
+)
+
+func EthToCometKey(hexPkey string) (ed25519.PrivKey, error) {
```

Comment:
The secret connection in cometbft didn't support sepk256k1, otherwise we could've gone full ethereum. I'll ask their team about that in a discussion.

Reply:
Otherwise, we need to do this one way conversion and find another way to tie the validator keys to the ethereum keys.